### PR TITLE
Fix the API to actually return the ACLs when requesting them

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: pkgdb2/api/packages.py [hunk 1]
```diff
@@ -905,22 +905,29 @@ def api_package_list(pattern=None):
         critpath = False
     elif critpath:
         critpath = True
-    acls = bool(flask.request.args.get('acls', False))
+    acls = flask.request.args.get('acls', False)
+    if str(orphaned).lower() in ['0', 'false']:
```
Comment (ralphbean) [inline pkgdb2/api/packages.py]: Do you mean ``acls`` instead of ``orphaned`` here?

Comment (pypingou) [inline pkgdb2/api/packages.py]: oups :]

Comment (ralphbean) [inline pkgdb2/api/packages.py]: for tech-debt-fighting week, one task could be to convert all little blocks like this to use an ``asbool`` function or something like that.

```python
acls = asbool(flask.request.args.get('acls', False))

def asbool(value):
    if value is None:
        return False
    if isinstance(value, bool):
        return value
    return str(value).lower() in ['1', 'true', 'yes', 'please']  # etc...

**Comment by ralphbean** [comment]
:+1:

File: pkgdb2/api/packages.py (hunk 1 above)
Comment (pypingou) [inline pkgdb2/api/packages.py]: :+1: for me

Comment (pypingou) [inline pkgdb2/api/packages.py]: One way around this is to create a Null object and to use it:

````python
class Null(Object):
   pass

branch = asbool(flask.request.args.get('branch', Null))

def asbool(value):
    if value is Null:
        return None
    if isinstance(value, bool):
        return value
    return str(value).lower() in ['1', 'true', 'yes', 'please']
````

**Comment by pypingou** [comment]
Thanks

**Pull request merged**